presentation/src/main/java/com/iluwatar/presentation/View.java

```diff
@@ -0,0 +1,159 @@
+package com.iluwatar.presentation;
+
+import java.awt.TextField;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import javax.swing.Box;
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JFrame;
+import javax.swing.JList;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Generates the GUI of albums.
+ */
+@Getter
+@Slf4j
+public class View {
+  /**
+   * the model that controls this view.
+   */
+  private final PresentationMod model;
+
+  /**
+   * the filed to show and modify title.
+    */
+  private TextField txtTitle;
+  /**
+   * the filed to show and modify the name of artist.
+   */
+  private TextField txtArtist;
+  /**
+   * the checkbox for is classical.
+   */
+  private JCheckBox chkClassical;
+  /**
+   * the filed to show and modify composer.
+   */
+  private TextField txtComposer;
+  /**
+   * a list to show all the name of album.
+   */
+  private JList<String> albumList;
+  /**
+   * a button to apply of all the change.
+   */
+  private JButton apply;
+  /**
+   * roll back the change.
+   */
+  private JButton cancel;
+
+  /**
+   * constructor method.
+   */
+  public View() {
+    model = new PresentationMod(PresentationMod.albumDataSet());
+  }
+
+  /**
+   * save the data to PresentationModel.
+   */
+  public void saveToPMod() {
+    LOGGER.info("Save data to PresentationModel");
+    model.setArtist(txtArtist.getText());
+    model.setTitle(txtTitle.getText());
+    model.setIsClassical(chkClassical.isSelected());
+    model.setComposer(txtComposer.getText());
+  }
+
+  /**
+   * load the data from PresentationModel.
+   */
+  public void loadFromPMod() {
+    LOGGER.info("Load data from PresentationModel");
+    txtArtist.setText(model.getArtist());
+    txtTitle.setText(model.getTitle());
+    chkClassical.setSelected(model.getIsClassical());
+    txtComposer.setEditable(model.getIsClassical());
+    txtComposer.setText(model.getComposer());
+  }
+
+  /**
+   * initialize the GUI.
+   */
+  public void createView() {
+    var frame = new JFrame("Album");
+    var b1 = Box.createHorizontalBox();
+
+    frame.add(b1);
+    albumList = new JList<>(model.getAlbumList());
+    albumList.addMouseListener(new MouseAdapter() {
+      @Override
+      public void mouseClicked(final MouseEvent e) {
+        model.setSelectedAlbumNumber(albumList.getSelectedIndex() + 1);
+        loadFromPMod();
+      }
+    });
+    b1.add(albumList);
+
+    var b2 = Box.createVerticalBox();
+    b1.add(b2);
+
+    txtArtist = new TextField();
+    txtTitle = new TextField();
+
+    final var widthTxt = 200;
+    final var heightTxt = 50;
```

Comment:
Declare the constants outside the method in the class section where you declare the member variables